pkg/sql/catalog/systemschema/system.go

```diff
@@ -403,10 +403,12 @@ CREATE TABLE system.statement_diagnostics_requests(
 	statement_fingerprint STRING NOT NULL,
 	statement_diagnostics_id INT8,
 	requested_at TIMESTAMPTZ NOT NULL,
+	min_execution_latency INTERVAL NOT NULL DEFAULT '0',
+	expires_at TIMESTAMPTZ NOT NULL DEFAULT now() + '100 years',
```

Comment:
Just curious but if the default value is meant to be really far into the future, should we consider making it a constant?

If the default is only used for the purpose of backfilling column values in existing rows, and all new rows will have `expired_at` set, as I suspect is the case, then perhaps don't specify a default value here in `systemschema`.
This makes the migration more complicated: you have to add the column either nullable or with the default (if you add it nullable you need to run an UPDATE query to backfill) and then subsequently remove the nullability or the default value from the column.

Perhaps it's not worth the trouble, though.